composer.json

```diff
@@ -148,6 +149,11 @@
         "symfony-web-dir": "web",
         "incenteev-parameters": {
             "file": "app/config/parameters.yml"
+        },
+        "heroku": {
+            "require": {
+                "ext-gd": "*"
```

Comment:
PHP dependencies on Heroku are actually installed without development dependencies ([ref](https://devcenter.heroku.com/articles/php-support#installation-of-dependencies)), so no tests.

Anyway I'm going to add the "ext-imagick" too :+1: